examples/blazor/Pages/Index.razor

```diff
@@ -0,0 +1,47 @@
+﻿@page "/"
+@using System.Timers;
+
+<div class="container">
+  <div class="col-md-8 offset-md-2">
+    <h4 class="mb-3">Contact</h4>
+    <form>
+      <div class="row">
+          <fluent-text-field @bind-value="FirstName" @bind-value:event="onchange">First Name</fluent-text-field>
```

Comment:
The syntax here looks more verbose than I'd have expected.

 * Does `@bind-value:event="onchange"` make any difference? I'd expect that to be the default anyway.
 * Can `@bind-value="FirstName"` be reduced to `@bind="FirstName"`? I'd expect that to be the same.

Reply:
I can confirm that `@bind="FirstName"` or ``@bind="@FirstName"` does not work, also no highlighting.
![image](https://user-images.githubusercontent.com/10981553/95930142-10b64b00-0dc6-11eb-9a0c-3806f5100936.png)